package.json

```diff
@@ -1,93 +1,85 @@
 {
-  "name": "giving-to-stanford",
+  "name": "ood_giving_site",
+  "version": "5.0.0",
   "private": true,
   "description": "Giving to Stanford",
-  "version": "4.2.6",
   "author": "Stanford University Office of Development",
-  "dependencies": {
-    "@fortawesome/fontawesome-free": "^5.15.4",
-    "algoliasearch": "^4.8.6",
-    "classnames": "^2.3.2",
-    "fuse.js": "^6.5.3",
-    "gatsby": "^4.24.8",
-    "gatsby-plugin-algolia": "^0.26.0",
-    "gatsby-plugin-google-tagmanager": "^4.3.0",
-    "gatsby-plugin-manifest": "^4.24.0",
-    "gatsby-plugin-netlify": "^4.1.0",
-    "gatsby-plugin-postcss": "^5.24.0",
-    "gatsby-plugin-react-helmet": "^5.24.0",
-    "gatsby-plugin-robots-txt": "^1.6.14",
-    "gatsby-plugin-sass": "^5.24.0",
-    "gatsby-plugin-sharp": "^4.24.0",
-    "gatsby-plugin-sitemap": "^5.24.0",
-    "gatsby-source-filesystem": "^4.24.0",
-    "gatsby-source-storyblok": "^4.2.1",
-    "gatsby-transformer-json": "^4.24.0",
-    "gatsby-transformer-sharp": "^4.24.0",
-    "netlify-plugin-contextual-env": "^0.3.0",
-    "netlify-plugin-submit-sitemap": "^0.4.0",
-    "netlify-plugin-vault-variables": "^2.0.0",
-    "path-browserify": "^1.0.1",
-    "process": "^0.11.10",
-    "query-string": "^7.1.3",
-    "react": "^17.0.2",
-    "react-autosuggest": "^10.1.0",
-    "react-dom": "^17.0.2",
-    "react-helmet": "^6.1.0",
-    "react-id-generator": "^3.0.2",
-    "react-instantsearch-dom": "^6.11.0",
-    "react-player": "^2.11.2",
-    "react-slick": "^0.29.0",
-    "react-spinners": "^0.11.0",
-    "react-table": "^7.7.0",
-    "sa11y": "^3.2.1",
-    "sanitize-html": "^2.7.3",
-    "slick-carousel": "^1.8.1",
-    "storyblok-js-client": "^4.1.6",
-    "storyblok-react": "^0.1.2",
-    "tabbable": "^5.2.1",
-    "trim": "^1.0.1"
-  },
-  "devDependencies": {
-    "decanter": "^6.2.19",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-prettier": "^4.2.1",
-    "local-ssl-proxy": "^1.3.0",
-    "node-vault": "^0.9.22",
-    "postcss": "^8.4.31",
-    "prettier": "2.3.2",
-    "react-refresh": "^0.13.0",
-    "toml": "^3.0.0"
-  },
   "keywords": [
-    "Stanford"
+    "nextjs",
+    "typescript",
+    "react",
+    "tailwindcss",
+    "Stanford",
+    "storyblok"
+  ],
+  "browserslist": [
+    "> 0.5%",
+    "last 2 versions",
+    "Firefox ESR",
+    "not dead"
```

Comment:
I'm in favor of changing these to `defaults` now as that seems to have a nice balance.

Reply:
IIFC these were actually the defaults (not sure if changed now), but we decided to write them out explicitly (I personally like that idea since we know for sure what we're committed to but i'm open to change) @sherakama